### DISCUSSION TITLE
GitHub.Network azure Resource provider getting stuck at registering

### DISCUSSION THREAD

**Discussion by yasirmohamed777** [community/community#125678] [Actions]
### Select Topic Area

Bug

### Body

Hi,
I am trying the [private networking for GitHub-hosted runners](https://docs.github.com/en/organizations/managing-organization-settings/configuring-private-networking-for-github-hosted-runners-in-your-organization) and the resource provider "GitHub.Network" in azure seems to be stuck registering for more than 24 hours. Anyone had similar issue?

**Comment by gpatounas** [comment]
This may not be the right place to bring up the discussion.
I had the same issue and was about to create a support case with microsoft but got distracted.
Now, after ~48 hours the provider is registered. 
I'm not sure if that is the expected behavior.

I can now move on to the next step and create the network settings resource which was showing `ResourceCreationValidateFailed` while the provider was still in the registering state.
